### PR TITLE
Update dolar-cripto-ar extension

### PR DESCRIPTION
**Pull request by koaladlt**
## Description

The extension was not fetching unless the user click on it. Now it is working as it should, fetching the quotes every 3 minutes.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/dolar-cripto-ar/CHANGELOG.md [hunk 1]
```diff
@@ -14,3 +14,4 @@
 - Introduced a dedicated `handleError` function for cleaner and more effective error handling.
 - Improved the state management and revalidation logic to prevent unnecessary fetch aborts.
 - Ensured smoother and more reliable fetching of currency data by refining the use of Abort Controllers.
+- Adjusted the interval logic to ensure currency quotes are refreshed every 3 minutes, enhancing data accuracy for users.
```
Comment (pernielsentikaer) [inline extensions/dolar-cripto-ar/CHANGELOG.md:17]: Could we create a new changelog entry?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

Could we create a new changelog entry?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/dolar-cripto-ar/CHANGELOG.md (hunk 1 above)
Comment (koaladlt) [reply extensions/dolar-cripto-ar/CHANGELOG.md:17]: Done!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**